src/v2/view-builder/views/TerminalView.js

```diff
@@ -3,11 +3,6 @@ import BaseView from '../internals/BaseView';
 import BaseForm from '../internals/BaseForm';
 
 const Body = BaseForm.extend({
-  title () {
-    // dont show title for terminal view
-    const msg = this.options.appState.get('terminal').message || {};
```

Comment:
Doesn't this break LEA ? Is the assumption here that API will never send `terminal`? From what I remember LEA API will still send terminal ?

Reply:
> From what I remember LEA API will still send terminal.

I don't think so after James work. @pradeepdewda-okta correct me wrong.

Reply:
ok I checked my old conversation with James and looks like we removed this from LEA as well. So we should be good!